CMakeLists.txt

```diff
@@ -31,6 +20,29 @@ option(USE_UBSAN "Enable UndefinedBehaviorSanitizer checks" OFF)
 option(USE_TSAN "Enable ThreadSanitizer checks" OFF)
 option(USE_MSAN "Enable MemorySanitizer checks" OFF)
 
+set(CXX_REQUIRING_OPTIONS
```

Comment:
Shouldn't be "CXX_REQUIRED_OPTIONS"?

Reply:
heh, I believe nope - is that some options are requiring CXX, not the other way around - maybe (but just maybe :D ) rename to `OPTIONS_REQUIRING_CXX` to be more clear 😉

Reply:
Renamed